changes/readme.md

```diff
@@ -0,0 +1,2 @@
+## Need to add some directions on how the updated process works
```

Comment:
Our other README files use all-caps for the file name. I'd like to keep that standard going forward